docs/dev/api.rst

```diff
@@ -66,6 +69,46 @@ The playlists API allows for the creation and editing of lists of recordings
    :include-empty-docstring:
    :undoc-static:
 
+Feedback API Endpoints
+^^^^^^^^^^^^^^^^^^^^^^
+These API endpoints allow to submit and retrieve feedback for a user's recordings
+
+.. autoflask:: listenbrainz.webserver:create_app_rtfd()
+   :blueprints: feedback_api_v1
+   :include-empty-docstring:
+   :undoc-static:
+
+Recording Recommendation API Endpoints
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+ListenBrainz uses collaborative filtering to generate recording recommendations,
+which may be further processed to generate playlists for users. These api endpoints
```

Comment:
I think we shouldn't specifically say that we use collaborative filtering here, or that we "might" generate playlists. Just say that we have a recommendation system that can generate similar recordings to those that a user has listened to. If we want to in the future we could add some pages to the developers section of the docs that explain how this works

Reply:
The endpoint uses collaborative filtering in its name so I think the intent might be to use different endpoints for different sources of recommendations in the future. I think In that case, it would be clearer if we mention CF here. also, @mayhem might have suggestions regarding this.

Reply:
good point, if the endpoint has CF in the name then explaining its use here is fine.